core/tests/integration/pass/core/records.ncl

```diff
@@ -1,141 +1,165 @@
 # test.type = 'pass'
```

Comment:
This field has a big diff because I formatted it with Topiary. You can ignore most of the changes but the very end, where I added a test for metadata keyword in field position.